tensorbay/opendataset/RarePlanesSynthetic/loader.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+#
+# Copytright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+
+"""Dataloader of RarePlanesSynthetic dataset."""
+
+import os
+from typing import Any, Callable, Dict, List, Tuple, Union
+
+import numpy as np
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import Classification, LabeledBox2D, PanopticMask
+from tensorbay.opendataset._utility import glob
+
+try:
+    from PIL import Image
+except ModuleNotFoundError:
+    from tensorbay.opendataset._utility.mocker import Image  # pylint:disable=ungrouped-imports
+try:
+    import xmltodict
+except ModuleNotFoundError:
+    from tensorbay.opendataset._utility.mocker import xmltodict  # pylint:disable=ungrouped-imports
+
+DATASET_NAME = "RarePlanesSynthetic"
+_ATTRIBUTES_GETTER: Dict[Tuple[int, ...], Callable[[str], Union[int, str, float]]] = {
+    (1, 3, 11, 13): float,
+    (14, 15): lambda x: x,
+    (12,): int,
+}
+
+
+def RarePlanesSynthetic(path: str) -> Dataset:
+    """`RarePlanesSynthetic <https://www.cosmiqworks.org/RarePlanes/>`_ dataset.
+
+    The file structure of RarePlanesSynthetic looks like::
+
+        <path>
+            images/
+                Atlanta_Airport_0_0_101_1837.png
+                ...
+            masks/
+                Atlanta_Airport_0_0_101_1837_mask.png
+                ...
+            xmls/
+                Atlanta_Airport_0_0_101_1837.xml
+                ...
+
+    Arguments:
+        path: The root directory of the dataset.
+
+    Returns:
+        Loaded :class:`~tensorbay.dataset.dataset.Dataset` instance.
+
+    """
+    root_path = os.path.abspath(os.path.expanduser(path))
+    dataset = Dataset(DATASET_NAME)
+    dataset.load_catalog(os.path.join(os.path.dirname(__file__), "catalog.json"))
+    category_ids = {
```

Comment:
I think we have a method named `get_category_to_index` in panoptic mask subcatalog, can we use it?